sandbox/demoapp/CreateChatView.qml

```diff
@@ -0,0 +1,215 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+import QtQml.Models 2.2
+
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+Page {
+    id: root
+    anchors.fill: parent
+    anchors.margins: 16
+    property ListModel contactsModel: null
+
+    header: RowLayout {
+        id: headerRow
+        width: parent.width
+        height: tagSelector.height
+        anchors.right: parent.right
+        anchors.rightMargin: 8
+
+        StatusTagSelector {
+            id: tagSelector
+            Layout.fillWidth: true
+            Layout.alignment: Qt.AlignVCenter | Qt.AlignLeft
+            Layout.leftMargin: 17
+            implicitHeight: 44
+            warningText: qsTr("5 USER LIMIT REACHED")
+            //simulate model filtering, TODO this
+            //makes more sense to be provided by the backend
+            //figure how real implementation should look like
+            property ListModel sortedList: ListModel { }
+            onTextChanged: {
+                sortedList.clear();
+                if (text !== "") {
+                    for (var i = 0; i < contactsModel.count; i++ ) {
+                        var entry = contactsModel.get(i);
+                        if (entry.name.toLowerCase().includes(text.toLowerCase())) {
+                            sortedList.insert(sortedList.count, {"publicId": entry.publicId, "name": entry.name,
+                                                  "icon": entry.icon, "isIdenticon": entry.isIdenticon,
+                                                  "onlineStatus": entry.onlineStatus});
+                            userListView.model = sortedList;
+                        }
+                    }
+                } else {
+                    userListView.model = contactsModel;
+                }
+            }
+        }
+
+        StatusButton {
+            implicitHeight: 44
+            enabled: (contactsModel.count > 0)
+            text: "Confirm"
+            onClicked: {
+                console.log("Confirm Join public chat");
+            }
+        }
+
+        StatusIconTabButton {
+            icon.name: "notification"
+            onClicked: {
+                print("create chat...");
+            }
+        }
```

Comment:
Cosmetics, but we can probably remove this one. The notification bell in the designs come from the `StatusChatToolBar` so I guess we 

1. Either use that here and draw the `StatusTagSelector` on top of it
2. Teach `StatusChatToolBar` to render components inside of it (like `StatusTagSelector`)
3. Just remove this notification thingy and don't assume that this is supposed to be the actual chat tool bar

Up to you!

Reply:
you are right, removed!